content/SCALE/SCALETutorials/Network/ConfigureIPv6.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Configuring IPv6"
+description: "Provides instructions configuring a network interface and other network settings for IPv6, and configuring an SMB or NFS share for IPv6."
+weight: 21
+tags:
+- network
+- interfaces
+- smb
+- nfs
+- ipv6
+---
+
+TrueNAS SCALE provides the option to configure either network interfaces using IPv4 or IPv6 addresses, or dual-stacking these protocols to allow for communication across the Internet where networks are only configured for one protocol.
```

Comment:
I'd leave out the part about dual-stacking.

Reply:
removed dual-stacking for now until we test and determine how to configure this. Another PR will address the "dual stacking" of these network in SCALE.